### ISSUE TITLE
Landing page for `machinery show --html`

### ISSUE THREAD

**Issue by tgoettlicher** [SUSE/machinery#1198]
If the user doesn't specify the system description name in the url (e.g. 'http://127.0.0.1:7585/'), the web server shows:
```
Sinatra doesn’t know this ditty.
Try this:

# in server.rb
class Server
  get '/' do
    "Hello World"
  end
end
```

Instead a landing page should be shown.

**Comment by vwallfahrer** [comment]
Fixing this issue will be postponed because it will be covered by my theses.

**Comment by promulo** [comment]
hi @tgoettlicher, not sure if this is valid anymore, at least I couldn't reproduce with current master.

**Comment by mauromorales** [comment]
Fixed in version 1.16.2 when the "landing page" was added.

Thanks @promulo !
